test/spec/grammar/compatibility_test.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-new, no-unused-expressions */
+const expect = require('chai').expect;
+const parse = require('../../../lib/dsl/poc/api').parse;
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+
+describe('Chevrotain Migration compatibility harness', () => {
+  describe('Can Parse all the existing JDL samples in the test folder', () => {
+    const samplesDir = path.resolve(__dirname, '../../test_files/');
+    const jdlSamples = _.filter(fs.readdirSync(samplesDir), file => _.endsWith(file, '.jdl'));
+
+    _.forEach(jdlSamples, (currSample) => {
```

Comment:
you needn't use lodash for looping

Reply:
I will also prefer the native forEach

Reply:
I will modify it.

Note that it does not really matter as this file is probably a **throwaway** file.
Once the migration is complete these JDL samples will be read as part of their own full test suites.